keyboards/clawsome/coupe/readme.md

```diff
@@ -1,5 +1,7 @@
 # Coupe
 
+![Coupe](https://www.clawboards.xyz/shop/coupe)
```

Comment:
The goal here is to have a direct photo of the keyboard here. (I really wish Squarespace URLs weren't so hideous.)

```suggestion
![Coupe](https://images.squarespace-cdn.com/content/v1/5ec961924eb3960db955f752/1591981988950-D64UDSNKPAT91WUA8RNC/ke17ZwdGBToddI8pDm48kDHPSfPanjkWqhH6pl6g5ph7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0mwONMR1ELp49Lyc52iWr5dNb1QJw9casjKdtTg1_-y4jz4ptJBmI9gQmbjSQnNGng/Coupe3.jpeg?format=640w)
```

Reply:
Maybe a "mirror" on imgur would be best.